caddy/plan.sh

```diff
@@ -1,13 +1,24 @@
 pkg_name=caddy
-pkg_description="Fast, cross-platform HTTP/2 web server with automatic HTTPS"
-pkg_upstream_url="https://caddyserver.com"
 pkg_origin=core
-pkg_version="v0.10.9"
+pkg_version="v0.11.0"
 pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
 pkg_license=("Apache-2.0")
-pkg_source="https://github.com/mholt/caddy/caddy"
+pkg_source="https://github.com/mholt/caddy/releases/download/v0.11.0/caddy_v0.11.0_linux_amd64.tar.gz"
```

Comment:
Caddy's pre-compiled binaries are licensed differently than it's source code. We'd need to confirm how the pre-compiled binaries available through github are licensed, otherwise anyone running Caddy would be in violation of Caddy's product licensing: https://caddyserver.com/products/licenses

Reply:
Yeah, caddyserver.com says binaries distributed from "this website", which is why i've pulled them from Github instead, which only appears to mention Apache 2.0 licensing. https://github.com/mholt/caddy/blob/master/LICENSE.txt

I'd argue that pulling anything from Github falls under the Apache 2.0 license.

Reply:
I'll follow up with them to confirm.

Reply:
The only reason I'm going to reach out and clarify is that they make several mentions of the source code specifically being Apache licensed. I sent an email to the email mentioned in their EULA (which you get when you download the .tar.gz from GitHub) to confirm and CC'd you.

Reply:
Perfect, thanks for CCing me 👍

Reply:
I still don't feel like they answered my question, but  ¯\_(ツ)_/¯

Reply:
@tduffield I didn’t get Czc’d on a response? Can you share?

Reply:
@predominant 

Me: 
> Does the EULA and the usage restrictions mentioned on https://caddyserver.com/products/licenses apply to binaries downloaded from https://github.com/mholt/caddy/releases, or just those downloaded from https://caddyserver.com/download? In either case, that is not clear and I might encourage clarification be made in the README.

Them:
> Thanks for reaching out. The EULA only applies to binaries from the website. Source code on github is under the Apache 2.0 license and has no such constraints.